src/rtkpos.c

```diff
@@ -2024,6 +2033,7 @@ static int relpos(rtk_t *rtk, const obsd_t *obs, int nu, int nr,
                 xp=x+K*v
                 Pp=(I-K*H')*P                  */
         trace(3,"before filter x=");tracemat(3,rtk->x,1,9,13,6);
+        sparse_mat_compress(H);
```

Comment:
Although this is in a loop, there is typically only one iteration. Also the sparse matrix does not appear to be used after this loop, so sparse_mat_free(H) could be mode up to just after this loop. The only consumer of the sparse matrix appears to be filter(). So it might be okay to omit the call to sparse_mat_compress() here too.